collect_app/src/main/java/org/odk/collect/android/activities/FormFillingActivity.java

```diff
@@ -1131,13 +1126,12 @@ private SwipeHandler.View createView(int event, boolean advancingPage) {
     private ODKView createODKView(boolean advancingPage, FormEntryPrompt[] prompts, FormEntryCaption[] groups) {
         odkViewLifecycle = new ControllableLifecyleOwner();
         odkViewLifecycle.start();
+        AudioPlayer audioPlayer = audioPlayerFactory.create(this, odkViewLifecycle);
+        audioPlayer.isLoading().observe(this, (isLoading) -> {
```

Comment:
Does this need to observe with the `odkViewLifecycle` instead?

Reply:
It wasn't using it https://github.com/getodk/collect/pull/6850/files#diff-0bf62cb249d6cb5715c94f28798547065602dc39e890993332bb6ed5fb42d88cL638, but yeah, I think it can now.